### DISCUSSION TITLE
[vcpkg] How to delete a certain port_triplet from the binary cache?

### DISCUSSION THREAD

**Discussion by Neumann-A** [microsoft/vcpkg#20740] [Q&A]
add answer to docs ;)

**Comment by linquize** [comment]
In `~/AppData/Local/vcpkg/archives` (Windows), `~/.cache/vcpkg/archives` (Linux)
```bash
find -type f -name '*.zip' -exec unzip -p -C {} CONTROL \;
```
will print the `CONTROL` files with ABI hashes, so that the user can decide which zip files to delete.
